pbrd/pbr_nht.c

```diff
@@ -870,9 +872,13 @@ static void pbr_nht_nexthop_update_lookup(struct hash_bucket *b, void *data)
 
 	pnhi.nhr = (struct zapi_route *)data;
 	pnhi.valid = 0;
```

Comment:
nit: maybe this could be 'false' also?

Reply:
Sure, I think this is the right move as we only care if any of the specified nexthops are valid (not how many)